src/cljs/ui/config.cljs

```diff
@@ -19,7 +19,7 @@
                     ["/new-analysis" :route/new-analysis]
                     ["/analysis-results" :route/analysis-results]
                     ["/documentation" :route/documentation]
-                    ]
+                    ["/map" :route/map]]
```

Comment:
I imagined that the map renderer is more of a separate application, that runs it's own process and accepts s3 URL as it's only argument, and renders the SpreadData (json) content? Unless this is for development purposes and we will split it up later?

Reply:
Yeah all that is just for development purposes, just so we can test the technology